go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/felixge/fgprof v0.9.2
 	github.com/florianl/go-nflog/v2 v2.0.1
 	github.com/fsnotify/fsnotify v1.5.1
+	github.com/gogo/googleapis v1.4.0
```

Comment:
can you switch the extauthz package to `google.golang.org/genproto/googleapis/rpc/status`? Using `gogo` is both incorrect in the first place and it introduces more dependencies which is not great